### DISCUSSION TITLE
No such file or directory

### DISCUSSION THREAD

**Discussion by fgaviriar** [ohmybash/oh-my-bash#337] [Q&A]
Why I get this message "`00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory`" when I want run for example:  `yarn run web:dev ` or `yarn web:dev `

**Comment by akinomyoga** [comment]
Oh My Bash (OMB) does not seem to have settings for `yarn` nor seem to include the strings like `XXhXXmXXs X/X` (I have searched for the codebase of OMB with the regular expression `.*h.*m.*s .*/.*`, but nothing relevant was found).

Does it still happen when you turn off OMB? You can temporarily turn off OMB by commenting out the following line in your `.bashrc`:

```bash
source "$OSH"/oh-my-bash.sh
```

Also, I'm not familiar with `yarn` as I don't use it for myself. How can we set up the `yarn` command? Could you describe the steps to set up and run `web:dev`? When I simply run `yarn web:dev`, it seems to produce the error `error Command "web:dev" not found.` How can we define `web:dev`? If that is something you have defined, could you provide me with the information on that, i.e., some minimal settings that reproduce the error message you get.
